src/SourceBuild/content/repo-projects/Directory.Build.props

```diff
@@ -69,10 +69,6 @@
     <EnvironmentVariables Include="DotNetBuildFromSourceFlavor=Product" />
     <EnvironmentVariables Include="DotNetPackageVersionPropsPath=$(PackageVersionPropsPath)" />
     <EnvironmentVariables Include="DotNetRestorePackagesPath=$(PackagesDir)" />
-    <EnvironmentVariables Include="DotNetBuildOffline=true" />
```

Comment:
Can this be removed as well - https://github.com/dotnet/dotnet/blob/main/repo-projects/Directory.Build.props#L160?